pages/common/spatial.md

```diff
@@ -0,0 +1,27 @@
+# spatial
+
+> A set of commands for managing and developing SpatialOS projects.
+
+- Build workers for local deployment on Unity on Windows:
+
+`spatial worker build --target=development --target=Osx`
```

Comment:
The target is osx here, but in the description you mention Windows. How does it work ?

Reply:
Mistake, will fix